readers/api/transport.go

```
@@ -140,6 +142,15 @@ func decodeList(_ context.Context, r *http.Request) (interface{}, error) {
 	if err != nil {
 		return nil, errors.Wrap(apiutil.ErrValidation, err)
 	}
```

Add check if aggregation if valid then there need to be a non empty to and from values

---

the non-empty should be interval in this case

---

if the interval is empty and aggregation is not it should return an error for example. I want to get the maximum value of data values over a 1hr period in the last 24 hours.